### ISSUE TITLE
Error when executing yarn

### ISSUE THREAD

**Issue by dappelt** [machinomy/machinomy-contracts#35]
I follow this guide to setup [machinomy](https://github.com/machinomy/machinomy/blob/master/docs/ganache-cli.md) with a local Etherum testnet. I am running into an error at step 2. Yarn gets hung up on "[1/4] Resolving packages ..." and the process crashes eventually.

```
$ cd machinomy-contracts
$ yarn
[1/4] 🔍  Resolving packages...

<--- Last few GCs --->

[26574:0x103000000]    62657 ms: Mark-sweep 1278.3 (1485.2) -> 1278.3 (1485.2) MB, 792.7 / 0.0 ms  allocation failure GC in old space requested
[26574:0x103000000]    63541 ms: Mark-sweep 1278.3 (1485.2) -> 1278.3 (1452.2) MB, 849.1 / 0.0 ms  last resort GC in old space requested
[26574:0x103000000]    64270 ms: Mark-sweep 1278.3 (1452.2) -> 1278.3 (1449.7) MB, 728.5 / 0.0 ms  last resort GC in old space requested


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 0x2d03ac125501 <JSObject>
    1: getExactVersionMatch
...
```

Any idea?

**Versions**
- node 9.7.1
- yarn 1.5.1
- osx 10.11.6

**Comment by ARyaskov** [comment]
We experienced same problem today. It happen due to using yarn 1.3.2+. Please, keep yarn on version 1.3.2.
Yes, we understand that depend on instrument instead of code modifying is bad habit, but as temporary solution we recommend you keep yarn on version 1.3.2

**Comment by ARyaskov** [comment]
@dappelt
It's been a weird yarnpkg 1.3.2+ bug - in case of many (5+) git repositories in dependencies in package.json, it was hanging.
We changed git repositories to npm-packages and 'yarn install' successfully complete now. 

**TL;DR - Contracts version 4.0.13 supports yarnpkg 1.3.2+**
